pkg/apis/v1alpha1/cluster_run_template.go

```diff
@@ -46,6 +46,9 @@ type RunTemplateSpec struct {
 	// the blueprint is applied. Templates support simple value
 	// interpolation using the $()$ marker format. For more
 	// information, see: https://cartographer.sh/docs/latest/templating/
+	// You should not define the namespace for the resource - it will automatically
+	// be created in the owner namespace. If the namespace is specified and is not
+	// the owner namespace, the resource will fail to be created.
```

Comment:
Shoulda mentioned this originally sorry- this part is only true for the YTT section. For the template, validation will stop users from specifying a namespace so the docs should reflect that. i.e. something like what you had before:
```
You cannot define the namespace for the resource. The resource will be created in the owner namespace.
```